### DISCUSSION TITLE
How do we know which version of Vue is compatible with the latest version of Quasar?

### DISCUSSION THREAD

**Discussion by BenJackGill** [quasarframework/quasar#16021] [General - Components / Directives / etc] [answered]
Right now I am wondering if I can safely upgrade to Vue 3.3, but also in the future I would like to know when I can safely upgrade to other versions or not.

My `package.json` looks like this:

```
{
...
  "dependencies": {
    "@quasar/extras": "^1.16.4",
    "quasar": "^2.12.1",
    "vue": "^3.0.0",
    ...
  },
  "devDependencies": {
    "@quasar/app-vite": "^1.4.3",
    ...
  },
  ...
}
```

So it says Vue has a semver of `"vue": "^3.0.0"` and based on that I think Quasar is compatible with any Vue version in the range `3.x.x`? Is that the correct interpretation?

And if not, what the correct way of knowing which version of Vue is currently compatible with Quasar?

**Comment by metalsadman** [comment, marked as answer]
quasar doesn't set it anymore, that's why it's in your deps so one can choose what v3 version they want. read release notes of the vue 3 version you are trying to upgrade to, and see if there are breaking changes.

**Reply by BenJackGill** [reply]
Sorry but I'm still a little confused. Does that mean Quasar is compatible with all of Vue `3.x.x`?

**Reply by smolinari** [reply]
If the Vue team stick to semver, then Quasar v2 should be compatible with all Vue v3 versions, yes.

Scott
